umap/utils.py

```diff
@@ -162,3 +163,53 @@ def merge_features(reference: list, latest: list, incoming: list):
         merged.append(item)
 
     return merged
+
+
+class UmapManifestStaticFilesStorage(ManifestStaticFilesStorage):
```

Comment:
Is this supposed to be registered in some setting ? I've tried the branch, but the statics have still their plain name.